src/rust/ensogl/lib/text/src/component/area.rs

```diff
@@ -197,13 +312,13 @@ impl Div {
 // === Line ===
 // ============
 
+/// Visual line representation.
 #[derive(Debug)]
 pub struct Line {
     display_object : display::object::Instance,
     glyphs         : Vec<Glyph>,
-    divs           : Vec<Div>,
+    divs           : Vec<f32>,
```

Comment:
They are not easily deriverable. Glyphs have internal offsets to keep them aligned etc. And you want to have really fast access to info where is a center and where is not. This is optimalization here. It would be especially useful in fonts which hare non monospaced - and you want to do binry search after pressing mouse in the text to know what is the offset there.

Reply:
call note: add docs why